### PR TITLE
chore(#7440): add implicit-arrow-linebreak

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#7440

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:7440-no-empty-function/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:7440-no-empty-function/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:7440-no-empty-function/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
I love this rule!

File: shared-libs/user-management/src/users.js [hunk 1]
```diff
@@ -745,18 +746,17 @@ const getUserDoc = (username, dbName) => db[dbName]
...
+  .all(['users', 'medic'].map(dbName => getUserDoc(name, dbName)))
+  .catch(error => {
+    if (error.status !== 404) {
+      return Promise.reject(error);
+    }
+    return Promise.reject({
+      status: 404,
+      message: `Failed to find user with name [${name}] in the [${error.db}] database.`
     });
+  });
```
Comment (garethbowen) [inline shared-libs/user-management/src/users.js:759]: Maybe it's just me, but I think the two changes in this file should have braces and an explicit return. Basically if the entire function can't fit on one line I think it's clearer to have the old style. There are a bunch of places in this PR that have this style. What do you think?

**Review by garethbowen — APPROVED** [review]
This matches my preference.

@jkuester This came up in a review the other day - what's your take on it?

File: shared-libs/user-management/src/users.js (hunk 1 above)
Comment (dianabarsan) [reply shared-libs/user-management/src/users.js:759]: I agree with this!

**Pull request merged**